core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/imagecatalog/ImageCatalogV4Endpoint.java

```diff
@@ -121,7 +121,8 @@ ImagesV4Response getImages(@PathParam("workspaceId") Long workspaceId,
             @QueryParam("stackName") String stackName,
             @QueryParam("platform") String platform,
             @QueryParam("runtimeVersion") String runtimeVersion,
-            @QueryParam("imageType") String imageType) throws Exception;
+            @QueryParam("imageType") String imageType,
+            @QueryParam("govCloud") boolean govCloud) throws Exception;
```

Comment:
Shouldn't it be more maintainable in the future if we introduce a variant field instead of the `govCloud` boolean?

Reply:
we just discussed